package.json

```diff
@@ -23,37 +23,37 @@
     "node": ">=0.10.0"
   },
   "dependencies": {
-    "URIjs": "1.14.1",
-    "bluebird": "2.3.10",
+    "URIjs": "^1.14.2",
```

Comment:
Yeah but if it is a new machine / new developer who just installed a new copy of the server?